docker/dev/Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN set -x && apt-get update -qq \
     libboost-filesystem-dev \
     libboost-log-dev \
     libboost-program-options-dev \
-    libboost-regex-dev
+    libboost-regex-dev \
+    curl
```

Comment:
Let's make this install line sorted alphabetically if we're already touching it.